### DISCUSSION TITLE
Thank you 🙏

### DISCUSSION THREAD

**Discussion by wrestlerdude** [YoungFellow-le/photoshop-22-linux#3] [General]
All I have to say. This worked for me flawlessly, the cc2019 was completely broken for me. Thank you so much!

Just a small questions however, do you know how to have DPI scaling akin to Windows? Seems like Photoshop internally only supports 100% and 200%.

**Comment by albakhtari** [comment]
You're most welcome!
> Just a small questions however, do you know how to have DPI scaling akin to Windows? Seems like Photoshop internally only supports 100% and 200%.

Maybe try looking in the _Graphics_ tab in `winecfg` (option 5 in the main menu)

**Reply by ikim23code** [reply]
@wrestlerdude have you been able to fix the DPI scaling? I had the same issue with Photoshop v19 and couldn't find a solution.

**Reply by albakhtari** [reply]
> Maybe try looking in the _Graphics_ tab in `winecfg` (option 5 in the main menu)

Have you tried this?

**Reply by wrestlerdude** [reply]
@ikim23code No unfortunately not. Even with the winecfg change nothing worked.

**Reply by ikim23code** [reply]
What I could find to work is to go to: Edit - Preferences - Interface - Ui Font Size and set it to Large, also check the "Scale UI to Font" option. It will not increase the font size but it will increase the size of the icons.
